scripts/globals/magic.lua

```diff
@@ -739,9 +754,12 @@ function calculateMagicBurst(caster, spell, target, params)
     end
 
     -- Obtain first multiplier from gear, atma and job traits
-    -- Add in bonus from BLM AMII merits (minimum 0, maximum 0.12 with 5/5 merits)
     modburst = modburst + (caster:getMod(dsp.mod.MAG_BURST_BONUS) / 100) + params.AMIIburstBonus;
 
+    if caster:isBehind(target) and caster:getMainJob() == dsp.job.NIN and caster:getMainLvl() >= 75 and caster:hasStatusEffect(dsp.effect.YONIN) then
```

Comment:
Remove main job and main level check. They are not needed. getMerit handles this.